core/src/bundle_stage.rs

```diff
@@ -304,16 +305,6 @@ impl BundleStage {
             &bank_start.working_bank,
         );
 
-        // qos rate-limited a tx in here, drop the bundle
-        if sanitized_bundle.transactions.len() != num_included {
```

Comment:
oh interesting, this looks like potential bug?

Reply:
wdym?

Reply:
the early return before metrics report was definitely a bug.